aws-modules/aws-s3/src/test/java/com/baeldung/s3/FileGeneratorUtil.java

```diff
@@ -0,0 +1,27 @@
+package com.baeldung.s3;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.UUID;
+import java.util.concurrent.ThreadLocalRandom;
+
+class FileGeneratorUtil {
+
+    private static final String MAGIC_WORD = "baeldung";
+    private static final ThreadLocalRandom RANDOM = ThreadLocalRandom.current();
+
+    public static File generate() {
+        String fileName = "test_" + UUID.randomUUID().toString() + ".txt";
+        File file = new File(fileName);
+        try (FileWriter writer = new FileWriter(file)) {
+            for (int i = 0; i < RANDOM.nextInt(20) + 1; i++) {
+                writer.write(MAGIC_WORD);
+            }
+        } catch (IOException exception) {
+            exception.printStackTrace();
```

Comment:
Can we throw this instead?

Reply:
I've removed the part of writing content to the file in the new implementation, it was unnecessary.